python/tidy/servo_tidy/tidy.py

```diff
@@ -601,11 +602,12 @@ def get_wpt_files(only_changed_files, progress):
 
 def check_wpt_lint_errors(files):
     wpt_working_dir = os.path.abspath(os.path.join(".", "tests", "wpt", "web-platform-tests"))
-    site.addsitedir(wpt_working_dir)
-    from tools.lint import lint
-    returncode = lint.lint(files)
-    if returncode:
-        yield ("WPT Lint Tool", "", "lint error(s) in Web Platform Tests: exit status {0}".format(returncode))
+    if os.path.isdir(wpt_working_dir):
```

Comment:
The https://github.com/servo/servo/issues/10639#issue-148776716 has the requirement:
> Don't attempt to import wpt tools.lint in https://github.com/servo/servo/blob/master/python/tidy/servo_tidy/tidy.py#L605 unless the tests/wpt/web-platform-tests directory exists in the current project.

I think this requirement is for external projects (which do not have wpt), if there is no wpt folder, then skip the `tools.lint` import.